tests/lib/managers/webhooks-test.js

```diff
@@ -0,0 +1,158 @@
+/**
+* @fileoverview Webhooks Manager Tests
+* @author ccheng
+*/
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake,
+	Webhooks,
+	webhooks,
+	testQS = { testQSKey: 'testQSValue' },
+	testParamsWithQs,
+	WEBHOOKS_ID = '1234',
+	MODULE_WEBHOOKS_PATH = '../../../lib/managers/webhooks';
+
+describe('Webhooks', function() {
+
+	beforeEach(function() {
+		// Setup Environment
+		boxClientFake = leche.fake(BoxClient.prototype);
+		testParamsWithQs = {qs: testQS};
+		// Register Mocks
+		mockery.enable({
+			useCleanCache: true
+		});
+		mockery.registerAllowable('http-status');
+		mockery.registerAllowable('util');
+		mockery.registerAllowable('../util/url-path');
+		mockery.registerAllowable('../util/errors');
```

Comment:
It think you're only using the url-path require, the rest of these can be removed